### PR TITLE
fix to use SkillzRandom.nextInt and update DLLs

### PR DESCRIPTION
**Pull request by stsamuel**

### CODE REVIEW

**Comment by stsamuel** [comment]
@mwseiden  @dzog

**Comment by aguirre-miriam** [comment]
:+1:

File: CaveRunner/Assets/CaveRun3D/Scripts/Skillz.js [hunk 1]
```diff
@@ -170,9 +170,10 @@ static class Skillz {
 		function Value() {
 			
 			var randomValue : float = 0;
+			var max = 999999999;
 			if(IsMatchInProgress()) {
 				var skillzRandom = SkillzClass.CallStatic.<AndroidJavaObject>("getRandom");
-				randomValue = skillzRandom.Call.<float>("nextFloat");
+				randomValue = skillzRandom.Call.<int>("nextInt", 0, max) * 1.0 / max;
```
Comment (mwseiden) [inline CaveRunner/Assets/CaveRun3D/Scripts/Skillz.js]: This needs to be changed in Unity wrapper

Comment (stsamuel) [inline CaveRunner/Assets/CaveRun3D/Scripts/Skillz.js]: Yes- I'll put up similar PR for wrapper

On Fri, Nov 20, 2015 at 5:58 AM, mwseiden <notifications@github.com> wrote:

> In CaveRunner/Assets/CaveRun3D/Scripts/Skillz.js
> <https://github.com/skillz/3d-cave-runner/pull/1#discussion_r45470619>:
>
> >  			if(IsMatchInProgress()) {
> >  				var skillzRandom = SkillzClass.CallStatic.<AndroidJavaObject>("getRandom");
> > -				randomValue = skillzRandom.Call.<float>("nextFloat");
> > +				randomValue = skillzRandom.Call.<int>("nextInt", 0, max) * 1.0 / max;
>
> This needs to be changed in Unity wrapper
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/skillz/3d-cave-runner/pull/1/files#r45470619>.
>



-- 
*Shawn Samuel, SVP of Engineering*
ssamuel@skillz.com <aparadise@skillz.com> | (617) 388-9171
*SKILL**Z*

Follow us on our blog: http://www.skillz.com/blog

CONFIDENTIALITY NOTICE: The materials in this electronic mail transmission
(including attachments) are private and confidential and are the property
of the sender, Skillz Inc. The information contained in the material is
intended only for the use of the named addressee(s). If you are not a named
addressee, be advised that any unauthorized disclosure, copying,
distribution or the taking of any action in reliance on the contents of
this material is strictly prohibited. If you have received this electronic
mail transmission in error, please immediately notify the sender by sending
an e-mail message and thereafter, destroy it immediately.

**Pull request merged**